### ISSUE TITLE
Fix missing font icons (plain/line version)

### ISSUE THREAD

**Issue by amacado** [devicons/devicon#248]
The following icons keep missing a plain/line version and therefore not beeing displayed properly on the gh-page (see screenshot below).

- [x] clojure
- [x] clojurescript
- [x] ember
- [x] mocha
- [x] npm
- [x] redux
- [x] yunohost
- [x] bitbucket
- [x] sourcetree
- [x] confluence
- [x] ssh
- [x] twitter

![image](https://user-images.githubusercontent.com/8781699/86516301-c402c180-be1f-11ea-94ee-f062504ba341.png)

**Comment by Thomas-Boi** [comment]
Everything has been fixed. There are two special cases: npm and ember. They only have 1 SVG and while those are colorful, they are also used as icons. This is possible because Icomoon can actually strip colors from SVGs.

**Comment by amacado** [comment]
I will reopen this issue, since some icons are still missing (updated the list on top of this issue)

**Comment by Thomas-Boi** [comment]
I made a mistake regarding the icons. Turns out that while some icons are missing svg files, others have the wrong data entered into the `devicon.json`.

For example,
![image](https://user-images.githubusercontent.com/43018778/102179302-bedc8080-3e5b-11eb-9cc2-e029a6504c06.png)


Twitter has a "plain" version listed. However, in the folder, there is only an "original" version. Updating it to "original" fix the issue
![image](https://user-images.githubusercontent.com/43018778/102179406-e895a780-3e5b-11eb-81f5-32348a68cd3d.png)


There's also an issue within the HTML file itself. In the `script.js`, the bottom two versions didn't exist before I fix it:
![image](https://user-images.githubusercontent.com/43018778/102179458-08c56680-3e5c-11eb-9e13-e74fcc7fa2a0.png)

This caused ember to not be displayed because ember only has an "original-wordmark" version:
![image](https://user-images.githubusercontent.com/43018778/102179572-39a59b80-3e5c-11eb-9ca9-fcb5756eb6fd.png)


So I went through every single icons that you have in the list. I have fixed all of them and they now show up in the `index.html`. I'll push my change in a new PR and link this issue to it.
